src/encoded/tests/test_status_trigger.py

```diff
@@ -491,13 +475,18 @@ def test_set_status_changed_paths_experiment_rep_and_in_progress_file(testapp, e
     # Set to in progress.
     testapp.patch_json(file['@id'], {'status': 'in progress'})
     res = testapp.patch_json(experiment['@id'] + '@@set_status?force_audit=true&update=true', {'status': 'released'}, status=200)
-    assert len(res.json_body['changed']) == 5
-    assert len(res.json_body['considered']) == 6
+    
+    # TODO- Figure why 2 is needed (will be addressed in a PYTEST-REFACTOR)
```

Comment:
ONE TODO

Reply:
@zoldello these asserts make sure set_status updates all of the relevant objects, for example when experiment status is changed, it will go into all objects in 
set_status_up = [
        'original_files',
        'replicates',
        'documents',
        'target',
    ]
and will update their statuses. If the sub-object is already in the status or it is the status that will not be changed, you will get more "considered" than "changed". So this is a test that should work, and the asserts are correct. Here is the output from running set_status here: 

> {"status": "success", "@type": ["result"], "changed": [["/files/ENCFF760VDE/", "in progress", "released"], ["/replicates/4505b306-d651-4bb8-b476-64e0844a2102/", "in progress", "released"], ["/libraries/ENCLB767EKH/", "in progress", "released"], ["/biosamples/ENCBS695AJX/", "in progress", "released"], ["/experiments/ENCSR120QIF/", "in progress", "released"]], "considered": [["/files/ENCFF760VDE/", "in progress", "released"], ["/sources/sigma/", "released", "released"], ["/replicates/4505b306-d651-4bb8-b476-64e0844a2102/", "in progress", "released"], ["/libraries/ENCLB767EKH/", "in progress", "released"], ["/biosamples/ENCBS695AJX/", "in progress", "released"], ["/experiments/ENCSR120QIF/", "in progress", "released"]]}

Reply:
@gabdank This was prematurely resolved so I missed it. I just pushed in a fix and awaiting a CI test-run